src/mappings/Cache.ts

```diff
@@ -120,4 +122,23 @@ export async function cachedController(accountAddress: string, event: SubstrateE
         controllersByStash[blockId] = bondedByAddress
         return bondedByAddress[accountAddress]
     }
+}
+
+export async function cachedEraIndex(event: SubstrateEvent): Promise<number> {
```

Comment:
For clarity I would name it StakingRewardEraIndex because we are caching it including staking reward delay